go-controller/pkg/util/gateway-cleanup.go

```diff
@@ -42,14 +42,14 @@ func GatewayCleanup(nodeName string, nodeSubnet *net.IPNet) error {
 	}
 	staticRouteCleanup(clusterRouter, nextHops)
 
-	// Remove the patch port that connects join switch to gateway router
-	_, stderr, err = RunOVNNbctl("--if-exist", "lsp-del", "jtor-"+gatewayRouter)
```

Comment:
can we use constants for these prefixes with better names? jtor, dtoj, rtoj? I don't know what dtoj is, a well named constant would help here.

Reply:
@trozet -- jtor is join-to-router, dtoj is distributed-router-to-join, rtoj is router-to-join. I will add comments above the code where i use it. This has been the case from the very beginning of this project. I just added `dtoj`. 

Will move to the constant prefixes in a separate PR.

I think they are concise and captures how they are connected in the logical topology. They are very helpful while debugging. Note that the logical switch port names can get too long if we start expanding these abbreviations. For example:

rtoj-GR_`hostname` and the `hostname` with FQDN could be 255 characters.

Reply:
@trozet yeah, that would be a good follow-up PR. Not that this is the right place for it, but these are "X to Y" where X and Y might be "router", "join", or "distributed router".

Reply:
@trozet BTW, I already have the following comment and I think its good enough.

`// jtod/dtoj - patch ports that connect the per-node join switch to distributed router`